kcc_cli/backlight_paths.py

```diff
@@ -18,3 +18,10 @@
         Position.EXTRA: '/sys/class/leds/system76::kbd_backlight/color_extra',
     }
 }
+
+SINGLE_BACKLIGHT_PATH = {
+    'brightness_path': '/sys/class/leds/system76::kbd_backlight/brightness',
+    'brightness_color': {
+        Position.CENTER: '/sys/class/leds/system76::kbd_backlight/color_left'
+    }
+}
```

Comment:
I would suggest changing `ONE_BACKLIGHT_PATH` to `SINGLE_BACKLIGHT_PATH`. 

I would suggest changing the current constant for this instead.

```suggestion
SINGLE_BACKLIGHT_VARIANT_PATH = {
    'brightness_path': '/sys/class/leds/system76::kbd_backlight/brightness',
    'brightness_color': {
        Position.CENTER: '/sys/class/leds/system76::kbd_backlight/color_left'
    }
}
```

Reply:
I am also wondering if there is a correlation between laptops with Coreboot? My Oryx Pro 6 has coreboot and its path to colour is different. Does the `darp5` have Coreboot? If it doesn't we could rename `SINGLE_BACKLIGHT_VARIANT_PATH` for `LEGACY_SINGLE_BACKLIGHT_PATH`. 

As far as my understanding goes on this, I believe that S76 is going to use Coreboot as much as possible since they have invested so into it. So I am guessing non-Coreboot MB are considered legacy on their end already.